pkg/block/gs/adapter.go

```diff
@@ -21,6 +21,8 @@ const (
 	delimiter    = "/"
 	partSuffix   = ".part_"
 	markerSuffix = ".multipart"
+
+	defaultPreSignedURLDuration = time.Minute * 15
```

Comment:
Same here - if we define it equally in 2 different places, let's move it up the packages stack.

Reply:
moved